src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1409,6 +1410,7 @@
   </data>
   <data name="FancyZones.ModuleTitle" xml:space="preserve">
     <value>FancyZones</value>
+	<comment>{Locked}</comment>
```

Comment:
This file seems to have gotten mangled by someone editing it in a text editor. It has mixed tabs and spaces. Please open it and re-save it using Visual Studio (and never edit it in a text editor again)